calculator-2/calculator.py

```diff
@@ -0,0 +1,60 @@
+"""CLI application for a prefix-notation calculator."""
+
+from arithmetic import *
+
+
+# Replace this with your code
+
+while True:
+    user_input = input("Enter your equation > ")
+    tokens = user_input.split(" ")
```

Comment:
This works fine, but you can omit the argument to user_input.split() and it will split by any whitespace, not just single spaces.